package.json

```diff
@@ -9,8 +9,8 @@
     "lint": "./node_modules/eslint/bin/eslint.js src/**/*.js",
     "test": "npm run lint && ./node_modules/jest-cli/bin/jest.js --coverage",
     "watch:test": "./node_modules/jest-cli/bin/jest.js --coverage --watch",
-    "coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
-    "travis": "npm install -g trevor && trevor"
+    "coverage": "./node_modules/jest-cli/bin/jest.js --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
```

Comment:
We're running the unit tests twice. Why? We've already generated the code coverage with the test command.

Reply:
Fixed.